pkg/local_object_storage/blobstor/delete_big.go

```diff
@@ -21,11 +21,13 @@ type DeleteBigRes struct{}
 // Returns any error encountered that did not allow
 // to completely remove the object.
 //
-// Returns ErrNotFound if there is no object to delete.
+// Returns apistatus.ObjectNotFound if there is no object to delete.
```

Comment:
It looks like an error here, but is really a type, I think we need to state this clearly.